test/BUILD

```diff
@@ -79,6 +79,23 @@ cc_binary(
     ],
 )
 
+# Test for resolving virtual include prefix
+cc_binary(
+    name = "virtual_include",
+    srcs = ["src/virtual_include.cc"],
+    deps = [":virtual_include_lib"],
+    includes = ["inc"],
+)
+
+# Header lib for virtual include
+cc_library(
+    name = "virtual_include_lib",
+    hdrs = glob(["inc/virtual_include_lib/*.h"]),
+    strip_include_prefix = "inc",
+    visibility = ["//visibility:public"],
+)
+
+
```

Comment:
I think the test for virtual include (strip_include_prefix) already exists at line 43